### ISSUE TITLE
[Doc] Rollups enhancements

### ISSUE THREAD

**Issue by hdhalter** [opensearch-project/documentation-website#792]
See https://github.com/opensearch-project/index-management/issues/408

**Comment by hdhalter** [comment]
Feature Review is 7/22

**Comment by kolchfa-aws** [comment]
This meta issue contains 4 sub-issues:
1. doc_count from source index -> in 2.2
2. allow dynamic target index in rollup searches -> in 2.2
3. allow rollup searches to multiple rollup indices -> in 2.2
4. allow query_string queries -> pushed to 2.3.

Issues #1-3 are documented. Waiting on a deployment to test these. Waiting for tech review in the meantime.

**Comment by Naarcha-AWS** [comment]
We'll  make a new issue for item 4 on the @kolchfa-aws's list. Closing for now.
